draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -274,7 +274,8 @@ This random token is placed in the RDATA as described in the rest of this sectio
 
 The TTL {{RFC1034}} for validation records SHOULD be short to allow recovering from potential misconfigurations. These records will not be polled frequently so caching or resolver load will not be an issue.
 
-Similarly, the SOA TTL (i.e. negative caching TTL) {{RFC2308}} for validation records SHOULD be short as well. Once the validation record is given to the user, the provider may start polling for its presence. The first attempts are likely to get an NXDOMAIN {{RFC2308}}, and if the NXDOMAIN is cached too long this would delay the validation.
+The service provider looking up these records may have to wait for the SOA TTL (negative caching TTL) to become visible if it has been previously queried. If the application user wants to make this visible more quickly they may need to work with the DNS administrator to see if they are willing to lower it, or ask the verifier to flush their DNS cache and re-querying it.
```

Comment:
```suggestion
The service provider looking up these records may have to wait for the SOA TTL (negative caching TTL) to become visible if it has been previously queried. If the application user wants to make this visible more quickly they may need to work with the DNS administrator to see if they are willing to lower it (which has implications across the entire zone), or ask the verifier to flush their DNS cache prior to re-querying it.

Verifiers MAY wish to use dedicated DNS resolvers configured with a low maximum negative caching TTL. 
```

Reply:
We might want to drop the "or ask the verifier to flush their DNS cache prior to re-querying it." since that seems unlikely to be practical.

Reply:
@enygren - your most recent updated text sounds good to me.

Reply:
@moonshiner - if this looks okay, I'll leave it to you to commit Erik's suggestion.

Reply:
I agree on dropping the "or ask the verifier to flush their DNS cache prior to re-querying it."